terraform/eval_diff.go

```diff
@@ -76,9 +76,9 @@ func (n *EvalCheckPlannedChange) Eval(ctx EvalContext) (interface{}, error) {
 	for _, err := range errs {
 		diags = diags.Append(tfdiags.Sourceless(
 			tfdiags.Error,
-			"Provider produced inconsistent final plan",
+			"Terraform produced inconsistent final plan",
 			fmt.Sprintf(
-				"When expanding the plan for %s to include new values learned so far during apply, provider %q produced an invalid new value for %s.\n\nThis is a bug in the provider, which should be reported in the provider's own issue tracker.",
+				"When expanding the plan for %s to include new values learned so far during apply, provider %q produced an invalid new value for %s.\n\nThis us a bug in either Terraform or the provider, please report it.",
```

Comment:
```suggestion
				"When expanding the plan for %s to include new values learned so far during apply, provider %q produced an invalid new value for %s.\n\nThis is a bug in either Terraform or the provider, please report it.",
```

Typo (us/is). Same above ☝️